battleship.py

```diff
@@ -199,6 +222,13 @@ def shipIsValid(grid, ship):
 Returns: None
 '''
 def placeShip(data):
+    if shipIsValid(data["userboard"],data["temporaryShip"]):
+        for each in data["temporaryShip"]:
+            data["temporaryShip"][each[0]][each[1]] = SHIP_UNCLICKED
+            data["NumberofuserShips"]+=1

```

Comment:
@rajeswari2498, let have a conversation on this 228th line of this function placeShip().

Reply:
Good job @rajeswari2498